docusaurus/docs/how-to-guides/plugin-security.md

```diff
@@ -0,0 +1,114 @@
+---
+id: plugin-security
+title: Plugin security
+description: A guide for secure plugin development.
+keywords:
+  - grafana
+  - plugins
+  - security
+  - safety
+  - privacy
+  - plugin security
+---
+
+# Secure plugin development
+
+Building secure plugins for Grafana is essential to protecting user data, maintaining trust, and avoiding vulnerabilities that could compromise your Grafana plugin. This guide covers best practices for secure plugin development across the backend and frontend.
+
+## General best practices
+
+### Security audits and reviews
```

Comment:
If we want people to use gosec it should become part of the default scaffolding, maybe as a step in GH actions workflow.

Reply:
it is - the validator runs in the release workflow, but we should mention that